lib/web_console/templates/console.js.erb

```diff
@@ -864,11 +864,23 @@ REPLConsole.prototype.scrollToBottom = function() {
   this.outer.scrollTop = this.outer.scrollHeight;
 };
 
-// Change the binding of the console
+/**
+ * Change the binding of the console
+ * Also updates a div to show that it has been changed.
```

Comment:
This may change, maybe we won't win a lot by changing the comment here.

Reply:
Makes sense, I'll revert the change here